src/cmds/openapi/index.ts

```diff
@@ -235,7 +237,10 @@ export default class OpenAPICommand extends Command {
 
       options.method = 'post';
       spinner.start('Creating your API docs in ReadMe...');
-      return fetch(`${config.get('host')}/api/v1/api-specification`, options).then(res => {
+      return fetch(`${config.get('host')}/api/v1/api-specification`, options, {
```

Comment:
I was very confused why you were passing `filePath` and `fileType` params to our API where they don't exist before realizing that `fetch()` here isn't `fetch` but a wrapper for `fetch`. 🫠

Reply:
😅 if you have an idea for an alternative pattern, I'm all ears!

Reply:
Aside from renaming the `fetch` wrapper to something else I think it's fine as-is.

Reply:
`apiFetch`? `readmeFetch`? `readmeApiFetch`? What do you think?

Reply:
`readmeAPIFetch`? And you could consolidate the `config.get('host')` stuff in there too so you don't need to repeat that everywhere.

could also dogfood an `api` SDK too 🥺

Reply:
we do some weird fetch stuff like overriding the base URL for proxies, extracting/logging fetch options prior to the request, and setting custom headers that aren't defined in the spec, is that all possible with `api`?

Reply:
ah i forgot about those :(

no its not possible with `api` yet

Reply:
I'm gonna rename this in a separate PR to keep this diff confined!